src/Gui/Forms/InitialPageInteractor.cs

```diff
@@ -29,14 +29,15 @@
 using System.ComponentModel.Design;
 using System.Linq;
 using System.Threading.Tasks;
+using System.IO;
 
 namespace Reko.Gui.Forms
 {
     public interface InitialPageInteractor : IPhasePageInteractor
     {
-        bool OpenBinary(string file);
-        bool OpenBinaryAs(string file, LoadDetails details);
-        bool Assemble(string file, Assembler asm);
+        bool OpenBinary(string file, string outputDir);
+        bool OpenBinaryAs(string file, string outputDir, LoadDetails details);
+        bool Assemble(string file, string outputDir, Assembler asm);
     }
```

Comment:
Same comment as before: the `OutputDirectory` should be a property internal to the `Project`.